src/gstreamer/launch/laptop_launch.py

```diff
@@ -0,0 +1,23 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+
+# Launches the joystick node and the gstreamer camera client node on the team laptop
+def generate_launch_description():
+    ld = LaunchDescription()
+
+    joystick_node = Node(
+        package="joy",
+        executable="joy_node",
+        parameters=["config/joy_node.yaml"],
+    )
+
+    GStreamer_node = Node(
```

Comment:
Here, instead of running a node, I think we need to run the rqt --force-discover CMD